scheds/rust/scx_lavd/src/main.rs

```diff
@@ -125,7 +125,7 @@ struct Opts {
     /// Run in monitoring mode. Show the specified number of scheduling
     /// samples every second.
     #[clap(long)]
-    monitor_sched_samples: Option<u64>,
+    monitor: Option<u64>,
```

Comment:
What lavd prints out is a bit different from stats printed out by other schedulers and the parameter's meaning is also different (number of samples instead of interval). Maybe it'd be useful to add more traditional statistics to lavd too which can then use the `--stats` and `--monitor` option rather than renaming sampling to take up the name.